ppl/archive/seekindex/seekindex.go

```diff
@@ -0,0 +1,88 @@
+package seekindex
+
+import (
+	"context"
+	"math"
+
+	"github.com/brimsec/zq/microindex"
+	"github.com/brimsec/zq/pkg/iosrc"
+	"github.com/brimsec/zq/pkg/nano"
+	"github.com/brimsec/zq/zbuf"
+	"github.com/brimsec/zq/zng"
+	"github.com/brimsec/zq/zng/resolver"
+)
+
+var Schema = []zng.Column{
```

Comment:
Maybe declare this in builder.go since that's where it's used? And maybe make it private?

Reply:
Sure